### PR TITLE
Added Azure Files troubleshooting details

### PR DESCRIPTION
**Pull request by ollypom**
### Proposed changes

Added Azure Files troubleshooting details when using the Dynamic Storage class with UCP.

### CODE REVIEW

File: ee/ucp/kubernetes/storage/use-azure-files.md [hunk 1]
```diff
@@ -238,6 +240,48 @@ spec:
 EOF
 ```
 
+### Troubleshooting
+
+When creating Persistent Volume Claims the volume may constantly stay in a
```
Comment (traci-morrison) [inline ee/ucp/kubernetes/storage/use-azure-files.md]: Suggestion: "When creating Persistent Volume Claims, the volume may constantly stay in a..."

File: ee/ucp/kubernetes/storage/use-azure-files.md [hunk 2]
```diff
@@ -238,6 +240,48 @@ spec:
...
+Kubernetes secret to store the Azure Files Storage Account Key.
+
+```
+Warning    ProvisioningFailed  7s (x3 over 37s)  persistentvolume-controller
+Failed to provision volume with StorageClass "standard": Couldn't create secret
+secrets is forbidden: User "system:serviceaccount:kube-system:persistent-volume-binder"
+cannot create resource "secrets" in API group "" in the namespace "default": access denied
+```
+
+To grant the `persistent-volume-binder` service account the relevant the RBAC
```
Comment (traci-morrison) [inline ee/ucp/kubernetes/storage/use-azure-files.md]: Suggestion: "To grant the `persistent-volume-binder` service account, the relevant the RBAC..."

File: ee/ucp/kubernetes/storage/use-azure-files.md [hunk 3]
```diff
@@ -173,12 +175,12 @@ After you create a Storage Class, you can use Kubernetes
 Objects to dynamically provision Azure Files Shares. This is done using
 Kubernetes Persistent Volumes Claims
```
Comment (traci-morrison) [inline ee/ucp/kubernetes/storage/use-azure-files.md]: Suggestion: Make "Persistent Volumes Claims" the text for the hyperlink. We don't refer to PVCs anywhere else in the document.

**Comment by ollypom** [comment]
Hey @traci-morrison , thanks for the feedback. Just pushed a new commit. PTAL :)

File: ee/ucp/kubernetes/storage/use-azure-files.md [hunk 4]
```diff
@@ -238,6 +239,48 @@ spec:
...
+Kubernetes secret to store the Azure Files Storage Account Key.
+
+```
+Warning    ProvisioningFailed  7s (x3 over 37s)  persistentvolume-controller
+Failed to provision volume with StorageClass "standard": Couldn't create secret
+secrets is forbidden: User "system:serviceaccount:kube-system:persistent-volume-binder"
+cannot create resource "secrets" in API group "" in the namespace "default": access denied
+```
+
+To grant the `persistent-volume-binder` service account, the relevant the RBAC
```
Comment (traci-morrison) [inline ee/ucp/kubernetes/storage/use-azure-files.md]: Should this be "...service account and the relevant RBAC permissions..."?

File: ee/ucp/kubernetes/storage/use-azure-files.md [hunk 5]
```diff
@@ -238,6 +239,48 @@ spec:
...
+NAME             STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+azure-file-pvc   Pending                                      standard       32s
+```
+
+If that is the case, the `persistent-volume-binder` service account does not
+have the relevant Kubernetes RBAC permissions. The storage account creates a
+Kubernetes secret to store the Azure Files Storage Account Key.
+
+```
+Warning    ProvisioningFailed  7s (x3 over 37s)  persistentvolume-controller
```
Comment (traci-morrison) [inline ee/ucp/kubernetes/storage/use-azure-files.md]: Is this a warning message for the user?

Comment (ollypom) [reply ee/ucp/kubernetes/storage/use-azure-files.md]: @traci-morrison Sorry, this is an output showing the error message from Kubectl. Added the Kubectl command in there now.

**Pull request merged**